pkg/standalone-utils/contracts/relayer/special/TetuShareValueHelper.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+import "@balancer-labs/v2-solidity-utils/contracts/openzeppelin/SafeERC20.sol";
+import "@balancer-labs/v2-solidity-utils/contracts/math/FixedPoint.sol";
+
+import "@balancer-labs/v2-interfaces/contracts/standalone-utils/ITetuSmartVault.sol";
+import "@balancer-labs/v2-interfaces/contracts/standalone-utils/ITetuStrategy.sol";
+
+contract TetuShareValueHelper {
+    using SafeERC20 for IERC20;
+    using FixedPoint for uint256;
+
+    // Since Tetu TokenRate is basically 1 + yield, 1 is a good default value when token is not fully initialized
+    uint256 private immutable _defaultRate = FixedPoint.ONE;
+
+    // Exposing these functions to make it easy to calculate rate on tests.
+    // Important: This function does not belong to Tetu's token vault!
+    function fromTetuAmount(uint256 wrappedAmount, ITetuSmartVault wrappedToken) external view returns (uint256) {
```

Comment:
If these are functions in production code only used for tests, could you put them in a mock?